.gitignore

```diff
@@ -22,6 +22,7 @@ sdist/
 var/
 *.egg-info/
 .installed.cfg
+Setup.exe
```

Comment:
To avoid include in source control the compiled installer. 
It is the default name that NSIS gives to new installers.